src/sentry/flags/docs/api.md

```diff
@@ -0,0 +1,86 @@
+# Flags API
+
+Host: https://sentry.io/api/0
+
+**Authors.**
+
+@cmanallen
+
+**How to read this document.**
+
+This document is structured by resource with each resource having actions that can be performed against it. Every action that either accepts a request or returns a response WILL document the full interchange format. Clients may opt to restrict response data or provide a subset of the request data.
+
+## Flag Logs [/organizations/<organization_id_or_slug>/flag-log/]
+
+- Parameters
+  - environment (optional, string) - Filter the result-set by environment.
+  - project (optional, string) - Filter the result-set by project-id.
+  - start (optional, string) - ISO 8601 format (`YYYY-MM-DDTHH:mm:ss.sssZ`)
+  - end (optional, string) - ISO 8601 format. Required if `start` is set.
+  - per_page (optional, number)
+    Default: 10
+  - offset (optional, number)
+    Default: 0
+
+### Browse Flag Logs [GET]
+
+Retrieve a collection of flag logs.
+
+**Attributes**
+
+| Column           | Type   | Description                                      |
+| ---------------- | ------ | ------------------------------------------------ |
+| action           | string | Enum of `created` or `modified`.                 |
+| environment      | string | The environment the change applies to.           |
+| flag             | string | The name of the flag changed.                    |
+| modified_at      | string | ISO-8601 timestamp of when the flag was changed. |
+| modified_by      | string | The user responsible for the change.             |
+| modified_by_type | string | Enum of `email`, `id`, or `name`.                |
+
+- Response 200
+
+  ```json
+  {
+    "data": [
+      {
+        "action": "created",
+        "environment": "production",
+        "flag": "my-flag-name",
+        "modified_at": "2024-01-01T05:12:33",
+        "modified_by": "2552",
+        "modified_by_type": "id"
+      }
+    ]
+  }
+  ```
+
+## Flag Log [/organizations/<organization_id_or_slug>/flag-log/<flag>/]
+
+### Fetch Flag Log [GET]
+
+Retrieve a single flag log instance.
+
+- Response 200
+
+  ```json
+  {
+    "data": {
+      "action": "modified",
+      "environment": "development",
+      "flag": "new-flag-name",
```

Comment:
Could we know the value too? 

If we knew `before` and `after` we could help debug by showing what it changed to

Reply:
Varies by provider.  I can add an optional field but this is not a trivial line of work.  I'd like to start with the minimum -- _something changed_ -- which the user will understand implicitly or go to their service provider to understand.